Reference/using-ioc.md

```diff
@@ -43,9 +43,16 @@ Here's an example of a custom global.asax class which initializes the IoC contai
 			//register all controllers found in your assembly
 			builder.RegisterControllers(typeof(MyApplication).Assembly);
 
-			//register umbraco MVC + webapi controllers used by the admin site
+			//register umbraco MVC + webapi controllers used in the back office
 			builder.RegisterControllers(typeof(UmbracoApplication).Assembly);
     			builder.RegisterApiControllers(typeof(UmbracoApplication).Assembly);
+			
+			//register webapi controllers for developer, users and forms
+			//treeviews used in the back office
```

Comment:
You don't need to mention 'treeviews' , trees are controllers. This should be re-worded like:

> you will need to register all MVC and webapi controllers found in other assemblies too. For example, if you are running in Umbraco Cloud you will have both Umbraco Forms and Diplo.TraceLogViewer installed and therefore you'll need to register their webapi controllers to, example: 

You should also leave this block of code commented out because people will just copy/paste this whole thing in and if they don't have Forms or Diplo.TraceLogViewer installed they will get compile time exceptions.